examples/app.py

```diff
@@ -62,7 +62,7 @@
         cd static
         npm install
         cd ..
-        npm install -g node-sass clean-css requirejs uglify-js
+        npm install -g node-sass@3.8.0 clean-css@3.4.24 requirejs uglify-js
```

Comment:
I think it would be better to not pin clean css and instead install clean-css-cli plus bump invenio-assets to latest beta release.